packages/core/src/components/label/label.tsx

```diff
@@ -92,6 +95,8 @@ export class Label implements ComponentInterface {
   }
 
   componentDidLoad() {
+    this.labelText = this.host.innerText;
```

Comment:
Waarom heb je dit in een `@State()` gezet?

In `startTruncate()` worden er DOM en Resize observers gemaakt. Daar zal je op moeten aansluiten of iets soortgelijks moeten maken, de sr-only toevoeging moet onafhankelijk van `truncate` werken.

Het probleem is namelijk als de inhoud nu anders wordt, blijft labelText gelijk:

![image](https://user-images.githubusercontent.com/9696261/188598031-145deec2-ab98-49de-bf3c-ee0070b21334.png)

Verder denk ik dat er een dubbele punt moet staan. Screen readers pauzeren dan heel even.

Reply:
Bij initiele render is `this.host.innerText` leeg. Door de `@State()` wordt het geupdate.

Reply:
Duidelijk, dank